docs/setup/discovery-server.md

```diff
@@ -0,0 +1,49 @@
+# Fast DDS Discovery Server
+
+This page gives instructions on how to use a Fast-DDS discovery server with the Create® 3 robot as a client.
+For more information on the Fast-DDS discovery server, please refer to the [eProsima documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/ros2/discovery_server/ros2_discovery_server.html).
+
+*We’ll use the term computer to refer to any SBC (single board computer), virtual machine, personal computer, etc.*
+
+## Configure Server
+
+1. Start the discovery server on the computer that you wish to use as your server by replacing <SERVER_IP> with the server’s IP.
+
+        fastdds discovery -i 0 -l <SERVER_IP> -p 11811
+   
+   ![](./data/fastdds.png)
+
+## Configure Create® 3 Robot as a Client
+1. Make sure your Create® 3 robot is connected to wifi. If it isn't, follow the directions [here](../provision/).
+
+!!! important
+    If you have enabled the discovery server previously, make sure it is disabled in the configuration settings **before** connecting it to a new wifi.*
```

Comment:
suggest changing end of sentence to "**before** connecting it to a new Wi-Fi network."